ecommerce/discounts_test.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from decimal import Decimal, getcontext
+
+from ecommerce.factories import ProductFactory, DiscountFactory
+from ecommerce.discounts import (
+    DiscountType,
+    PercentDiscount,
+    FixedPriceDiscount,
+    DollarsOffDiscount,
+)
+
+pytestmark = [pytest.mark.django_db]
+
+
+@pytest.fixture()
+def products():
+    return ProductFactory.create_batch(5)
+
+
+@pytest.fixture()
+def discounts():
+    return DiscountFactory.create_batch(10)
+
+
+def test_discount_factory_generation(discounts):
```

Comment:
Could you please add comments to each test, mentioning what thei are testing?

Reply:
Added these in 20db5b212d945e1d53af7a820d3986145a0a8927